ide/team.commons/src/org/netbeans/modules/bugtracking/commons/LinkButton.java

```diff
@@ -119,10 +119,10 @@ public LinkButton() {
     }
     
     public void setColors(Color linkColor, Color linkInFocusColor, Color mouseOverLinkColor, Color visitedLinkColor) {
-        this.linkInFocusColor = linkInFocusColor;
-        this.linkColor = linkColor;
-        this.mouseOverLinkColor = mouseOverLinkColor;
-        this.visitedLinkColor = visitedLinkColor;
+        linkInFocusColor = linkInFocusColor;
+        linkColor = linkColor;
+        mouseOverLinkColor = mouseOverLinkColor;
+        visitedLinkColor   = visitedLinkColor;
```

Comment:
Seems wrong and unrelated to the described issues

Reply:
Well, maybe not unrelated but wrong.

Reply:
Short answer something like `LinkButton.linkInFocusColor = linkInFocusColor` for these four.

Long answer: If you simply remove the `this.` you essentially assign the method parameter to itself. Highlighting in NetBeans Editor will show that as well.

Reply:
No problem. It's now changed.. Thanks for the feedback..